src/gseproject/EXEplorer/models.py

```diff
@@ -1,7 +1,41 @@
+from django.conf import settings
 from django.db import models
-from django.contrib.auth.models import User
 # Create your models here.
 
-class UserProfile(models.Model):
-    user = models.OneToOneField(User, on_delete=models.CASCADE)
-    coins = models.IntegerField(default=0)
\ No newline at end of file
+class Player(models.Model):
+    user = models.CharField(settings.AUTH_USER_MODEL, on_delete = models.CASCADE, verbose_name = "User", related_name = "players")
+    playerID = models.AutoField(primary_key = True, verbose_name = "Player ID")
+
+    def __str__(self):
+        return f"{self.user.username} - Player {self.playerID}"
+
+
+class Tools(models.Model):
+    toolsID = models.AutoField(primary_key=True, verbose_name = "Tools ID")
+    toolsName = models.CharField(max_length=100, verbose_name = "Tools Name")
+    toolsDescription = models.TextField(verbose_name = "Tools Description")
+
+    def __str__(self):
+        return self.toolsName
+
+
+class Currency(models.Model):
+    player = models.ForeignKey(Player, on_delete = models.CASCADE, verbose_name = "Player", related_name = "currencies")
+    cherry = models.PositiveIntegerField(default = 0, verbose_name = "cherry")
+    
+    def __str__(self):
+        return f"{self.cherry} cherries - {self.player}"
+
+
+
+class Tree(models.Model):
+    player = models.ForeignKey(Player, on_delete = models.CASCADE, verbose_name = "Player", related_name = "trees")
+    tree = models.PositiveIntegerField(default = 0, verbose_name = "tree")
```

Comment:
What does the field tree represent?

Reply:
We plant trees in game. Tree represents 4 states (seed, sprout ......)